apps/portals/src/portal-components/DetailsPage/DetailsPageTabs.tsx

```diff
@@ -40,6 +42,16 @@ const DetailsPageTabs: React.FunctionComponent<DetailsPageTabsProps> = (
       </Switch>
       <div className="tab-groups">
         {tabConfigs.map((tab, index) => {
+          if (tab.hideIfColumnValueUndefined) {
+            if (rowValues && headers) {
+              const colIndex = headers.findIndex(h => h.name == tab.hideIfColumnValueUndefined)
+              if (!rowValues[colIndex]) {
```

Comment:
I'd expect us to filter nullish values rather than falsy values here. Since these are always strings or null, I think the only case where this will make a difference is the empty string. Does Synapse ever give us empty strings, and should we treat those like `null`?
```suggestion
              if (rowValues[colIndex] == null) {
```